public/js/icinga/behavior/navigation.js

```diff
@@ -30,6 +36,32 @@
          */
         this.active = null;
 
+        /**
+         * The co-ordinates of the triangle formed from the previous cursor position
+         * and the left corners of the flyout
+         *
+         * @type {Array}
+         */
+        this.coordinates = new Array(3);
```

Comment:
*coordinates* is not really a good name. Without looking at the doc block, it reads as *navigation coordinates*, which is obviously wrong. Please find an alternative. (Without the term *coordinates*)